lib/src/project.rs

```diff
@@ -29,9 +29,9 @@ fn deserialize_project_name<'de, D>(deserializer: D) -> Result<ProjectName, D::E
 impl ProjectName {
     pub fn is_valid(hostname: &str) -> bool {
         fn is_valid_char(byte: u8) -> bool {
-            (byte >= b'a' && byte <= b'z')
-                || (byte >= b'A' && byte <= b'Z')
-                || (byte >= b'0' && byte <= b'9')
+            (b'a'..=b'z').contains(&byte)
```

Comment:
I wonder why. This is definitely more elegant, but is probably slower and takes up more memory.

Reply:
I believe this was just a correction of [this Clippy lint.](https://rust-lang.github.io/rust-clippy/master/#manual_range_contains) You raise an interesting point about efficiency - I wonder if the compiler is clever enough to optimise it to be equivalent to the former under the hood or not.